From 3c6f6a572d4cf01a29db436929e435e0499387df Mon Sep 17 00:00:00 2001
From: Oliver Tappe <zooey@hirschkaefer.de>
Date: Mon, 16 Sep 2013 22:37:49 +0200
Subject: Bypass checks for autoconf git checkout

* As part of test preparations, autoconf's build system tries to determine
  if it lives in a checkout of the autoconf repository by looking if .git
  exists. In our case, .git exists, so the test preparation code tries
  to checkout a git submodule, which fails as the .git refers to the
  git repository created by haikuporter, not the autoconf repository.
  Circumvent this by causing the check for .git to always fail.

diff --git a/maint.mk b/maint.mk
index 1c7af03..41e4030 100644
--- a/maint.mk
+++ b/maint.mk
@@ -1294,7 +1294,7 @@ endef
 
 .PHONY: no-submodule-changes
 no-submodule-changes:
-	if test -d $(srcdir)/.git; then					\
+	if test -d $(srcdir)/.gitFOOBAR; then					\
 	  diff=$$(cd $(srcdir) && git submodule -q foreach		\
 		  git diff-index --name-only HEAD)			\
 	    || exit 1;							\
@@ -1312,7 +1312,7 @@ submodule-checks ?= no-submodule-changes public-submodule-commit
 # cannot be built from a fresh clone.
 .PHONY: public-submodule-commit
 public-submodule-commit:
-	$(AM_V_GEN)if test -d $(srcdir)/.git; then			\
+	$(AM_V_GEN)if test -d $(srcdir)/.gitFOOBAR; then			\
 	  cd $(srcdir) &&						\
 	  git submodule --quiet foreach test '$$(git rev-parse $$sha1)'	\
 	      = '$$(git merge-base origin $$sha1)'			\
-- 
1.8.3.4


From 66a0514f74a0e29994e3d42a10b5f4b1162aaa27 Mon Sep 17 00:00:00 2001
From: Ingo Weinhold <ingo_weinhold@gmx.de>
Date: Wed, 2 Oct 2013 16:18:53 +0200
Subject: Install prefix is now /boot/system


diff --git a/INSTALL b/INSTALL
index d60e29a..f386627 100644
--- a/INSTALL
+++ b/INSTALL
@@ -247,10 +247,10 @@ directory contains several dysfunctional programs; working variants of
 these programs are available in '/usr/bin'.  So, if you need '/usr/ucb'
 in your 'PATH', put it _after_ '/usr/bin'.
 
-   On Haiku, software installed for all users goes in '/boot/common',
+   On Haiku, software installed for all users goes in '/boot/system',
 not '/usr/local'.  It is recommended to use the following options:
 
-     ./configure --prefix=/boot/common
+     ./configure --prefix=/boot/system
 
 Specifying the System Type
 ==========================
diff --git a/doc/install.texi b/doc/install.texi
index 4c2ffb5..c1a987b 100644
--- a/doc/install.texi
+++ b/doc/install.texi
@@ -287,11 +287,11 @@ directory contains several dysfunctional programs; working variants
 of these programs are available in @code{/usr/bin}.  So, if you need
 @code{/usr/ucb} in your @env{PATH}, put it @emph{after} @code{/usr/bin}.
 
-On Haiku, software installed for all users goes in @file{/boot/common},
+On Haiku, software installed for all users goes in @file{/boot/system},
 not @file{/usr/local}.  It is recommended to use the following options:
 
 @example
-./configure --prefix=/boot/common
+./configure --prefix=/boot/system
 @end example
 
 @node System Type
-- 
1.8.3.4

